### PR TITLE
French localization for Broadcasts 2

### PR DESCRIPTION
**Pull request by flowbe**
Here is an update for the French localizations with the new strings for Broadcasts 2

### CODE REVIEW

File: fr.lproj/Localizable.strings [hunk 1]
```diff
@@ -158,3 +157,55 @@
...
+"EDIT_STATION_WINDOW_URL_FIELD_LABEL" = "adresse";
+"EDIT_STATION_WINDOW_IMAGE_FIELD_LABEL" = "illustration";
+
+"EDIT_STATION_WINDOW_IMAGE_BUTTON_CHOOSE" = "Choisir une image…";
+"EDIT_STATION_WINDOW_LIMIT_REACHED" = "La limite de la bibliothèque d’essai est atteinte. Veuillez procéder à une mise à niveau pour ajouter d'autres stations.";
+
+"TOOLBAR_PREVIOUSBUTTON_TOOLTIP" = "Précédent";
+"TOOLBAR_NEXTBUTTON_TOOLTIP" = "Suivant";
+
+"ALERT_UPGRADE" = "Mettre à niveau";
```
Comment (steventroughtonsmith) [inline fr.lproj/Localizable.strings:211]: Just to confirm — 'Upgrade' in this context is about upgrading from the free to paid version, via in-app purchase. Is this still how you would word this, and EDIT_STATION_WINDOW_LIMIT_REACHED above?

**Comment by steventroughtonsmith** [comment]
Thanks for contributing! Could you by any chance do one more string? Added to the bottom of the latest copy of the English version is this:

`"MENU_APPLICATION_UPGRADE" = "Upgrade to Full Version…";`

File: fr.lproj/Localizable.strings (hunk 1 above)
Comment (TheFlow95) [reply fr.lproj/Localizable.strings:211]: Yes! 😊

**Comment by TheFlow95** [comment]
No problem, I added it in my latest commit!

**Pull request merged**